<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualManipulationWidget</class>
 <widget class="QWidget" name="ManualManipulationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="interactionGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Interaction</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="interactTranslate">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Translate by either left-clicking and dragging the central handle, or by middle-clicking and dragging the data source.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Translate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="interactRotate">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotate by left-clicking a face and dragging it.&lt;/p&gt;&lt;p&gt;Rotation interactions are performed about the center of the data source.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="interactScale">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rescale individual axes by left-clicking the handles on the faces and dragging them.&lt;/p&gt;&lt;p&gt;Rescale with fixed aspect ratio by right-clicking the data source and moving the mouse.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="transformGridLayout">
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="scaleZ">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000010000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="rotateX">
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="zLabel">
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="rotateZ">
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="rotateY">
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="scaleX">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000010000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="scaleY">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000010000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="scaleLabel">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="shiftLabel">
       <property name="text">
        <string>Shift:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rotateLabel">
       <property name="text">
        <string>Rotate:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="shiftX">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="shiftY">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="shiftZ">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="referenceDataGroup">
     <property name="title">
      <string>Reference Data</string>
     </property>
     <layout class="QGridLayout" name="referenceGridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="selectedReferenceData">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If a reference data source is selected, it will be visually moved so that its center matches the center of this dataset. This will assist with aligning the volumes.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Once the operation is finished, the reference data source will be moved back to its original position.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="referenceDataLabel">
        <property name="text">
         <string>Data Source:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="alignVoxelsWithReference">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, the voxels of this data source will be cropped, padded, and subsampled (if the spacings are different) so that the voxels will be aligned with that of the reference data source.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Align voxels with reference?</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="referenceDataValuesWidget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="referenceValuesGridLayout">
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="referenceSpacingZ">
             <property name="decimals">
              <number>5</number>
             </property>
             <property name="minimum">
              <double>0.000010000000000</double>
             </property>
             <property name="maximum">
              <double>1000000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="referenceSpacingX">
             <property name="decimals">
              <number>5</number>
             </property>
             <property name="minimum">
              <double>0.000010000000000</double>
             </property>
             <property name="maximum">
              <double>1000000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QDoubleSpinBox" name="referenceSpacingY">
             <property name="decimals">
              <number>5</number>
             </property>
             <property name="minimum">
              <double>0.000010000000000</double>
             </property>
             <property name="maximum">
              <double>1000000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Spacing:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Shape:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="referenceShapeX">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="referenceShapeY">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="referenceShapeZ">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>interactTranslate</tabstop>
  <tabstop>interactRotate</tabstop>
  <tabstop>interactScale</tabstop>
  <tabstop>shiftX</tabstop>
  <tabstop>shiftY</tabstop>
  <tabstop>shiftZ</tabstop>
  <tabstop>rotateX</tabstop>
  <tabstop>rotateY</tabstop>
  <tabstop>rotateZ</tabstop>
  <tabstop>scaleX</tabstop>
  <tabstop>scaleY</tabstop>
  <tabstop>scaleZ</tabstop>
  <tabstop>selectedReferenceData</tabstop>
  <tabstop>alignVoxelsWithReference</tabstop>
  <tabstop>referenceSpacingX</tabstop>
  <tabstop>referenceSpacingY</tabstop>
  <tabstop>referenceSpacingZ</tabstop>
  <tabstop>referenceShapeX</tabstop>
  <tabstop>referenceShapeY</tabstop>
  <tabstop>referenceShapeZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
